django/db/backends/base/client.py

```diff
@@ -21,6 +21,6 @@ def settings_to_cmd_args_env(cls, settings_dict, parameters):
 
     def runshell(self, parameters):
         args, env = self.settings_to_cmd_args_env(self.connection.settings_dict, parameters)
-        if env:
+        if env is not None:
```

Comment:
I think this is good, but wonder if we can also change to the following in `django.db.backends.postgresql.client.DatabaseClient.settings_to_cmd_args_env()`:

```python
return args, (env or None)
```

This seems to fit with the expectation that `None` is returned when nothing is being added to the environment.